From 7bcf770fdf667082bcc1d821c1ee34ffd8ae1732 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 11 May 2022 15:35:50 +0100
Subject: [PATCH] SQUASH: overlays: arducam-pivariety: Strip leading
 zero

Newer versions of dtc warn about node addresses with leading zeroes.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/arducam-pivariety-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/arducam-pivariety-overlay.dts
+++ b/arch/arm/boot/dts/overlays/arducam-pivariety-overlay.dts
@@ -13,7 +13,7 @@
 			#size-cells = <0>;
 			status = "okay";
 
-			arducam_pivariety: arducam_pivariety@0c {
+			arducam_pivariety: arducam_pivariety@c {
 				compatible = "arducam,arducam-pivariety";
 				reg = <0x0c>;
 				status = "okay";
